packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.scss

```diff
@@ -84,3 +84,18 @@
   width: 58px;
   height: 58px;
 }
+
+.text-footer div {
```

Comment:
To eliminate repeating ourselves, we could add all of the common styles between subplat and fxa layouts into the global file.

In the global:
```
.text-footer div {
  @extend .font-sans;
  text-align: center !important;
  color: global.$grey-400 !important;
}

.text-footer-automatedEmail {
  @extend .text-footer;
  div {
    @extend .mb-3;
    @extend .mt-10;
  }
}
```
Then in fxa layout:

```
.text-footer div {
  @extend .text-xs;
}
```

and in subplat layout:
```
.text-footer div {
  @extend .text-sm;
}